kr/web.php

```diff
@@ -0,0 +1,95 @@
+<?php
+return [
+    'leaderboard' => '리더 보드',
+    'tournament' => '토너먼트',
+    'login' => '로그인',
+    'register' => '가입',
+    'myProfile' => '내 프로필',
+    'myImprovement' => '나의 기록들',
+    'settings' => '설정',
+    'logout' => '로그 아웃',
+    
+    /* Leaderboard */
+    'name' => '이름',
+    'time' => '시각',
+    'blocks' => '블록',
+    'PPS' => '초당 사용하는 블럭 갯수',
+    'finesse' => '피네스',
+    'date' => '날짜',
+    'replay' => '다시보기',
+    'place' => '공간',
+    'nli' => '로그인 실패',
+    'today' => '오늘',
+    'lastWeek' => '지난주',
+    'lastMonth' => '지난 달',
+    'lastYear' => '작년',
+    'allTime' => '모든 기록들',
+    'bestPlayers' => '최고의 선수들',
+    'bestGames' => '최고의 게임',
+    'bestSubs' => '가장 빠른 시간',
+    'latestGames' => '이전 게임들',
+    'leastBlocks' => '적은 블럭수',
+    'myTimes' => '나의 기록들',
+    
+    /* Replay */
+    'load' => '불러오기',
+    'play' => '게임하기',
+    'pause' => '멈춤',
+    'showSeg' => '구분해서 보기',
+    'fastestSeg' => '가장 빠른 구간',
+    'slowestSeg' => '가장 느린 구간',
+    'speedChart' => '속도 차트',
+    'moreStats' => '스탯 더보기',
+    
+    /* PW change */
+    'pwChange' => '비밀번호 변경',
+    'newPw' => '새 비밀번호',
+    'againPw' => '새 비밀번호 재입력',
+    'applyChange' => '변경하기',
+    
+    /* Profile */
+    'ownProfile' => '공개 프로필',
+    'bestTimes' => '가장 좋은 시간',
+    'allTimes' => '모든 스프린트 기록 보기',
+    'userImpr' => '더 많은 게임 내용',
+    'gameStats' => '게임 내용',
+    'latestGames' => '지난 게임',
+    'noGames' => '사용자가 아직 게임을 하지 않았습니다.',
+    'gamesPlayed' => '게임 플레이',
+    'totalTime' => '플레이 시간',
+    'linesSent' => '보냄 ',
+    'linesRec' => '받음',
+    'placedBlocks' => '사용한 블럭 갯수',
+    'maxAPM' => '최대 공격 속도',
+    'maxREN' => '최대 콤보',
+    'longestGame' => '가장 긴 게임',
+    'totalB2B' => '모든 백투백',
+    'mostSent' => '가장 많이 보낸 공격',
+    'hours' => '시간',
+    'minutes' => '분',
+    
+    /* Stats */
+    'improvementStats' => '더 많은 내용',
+    //keep ':이름'
+    'improvementInfo' => '라인레이스 시간 (Y 축) vs 게임 한 날짜 (X 축). 강조된 선은 이름 또는 개인 기록을 표시합니다. 커서를 통해 임의대로 세부 정보를 확인 할 수 있습니다.',
```

Comment:
This line should contain `:name` place holder. This gets then replaced with player's name (if you look how this line is used on the site you will see your username mentioned in it).

The comment above should say `//keep ':name'`, this string has a special purpose and is not translated.

Reply:
ah okay. I fixed it back to //keep ':name'  do i have to issue another pull request or is it automatically updated?

Reply:
The translation itself at the line 74 should contain `:name`. It's where the username gets inserted, the comment is there just so it doesn't get deleted in editing.

After you make the change the pull request is updated automatically, no need for another one.